src/java/ru/javarush/cryptoanalyser/Main.java

```diff
@@ -0,0 +1,169 @@
+                            "%n3. With statistical analysis");
+                    currentLine = scanner.nextLine();
+
+                    if ("1".equals(currentLine)) {
+                        withKey = true;
+
+                        System.out.println("Enter integer key for decrypt");
+                        while (!"exit".equals(currentLine)) {
+                            currentLine = scanner.nextLine();
+
+                            if (Validator.isNumeric(currentLine)) {
+                                key = Integer.parseInt(currentLine);
+                                currentLine = "exit";
+                                break;
+                            } else {
+                                System.out.println("Wrong key format");
+                                currentLine = scanner.nextLine();
+                            }
+                        }
+                    } else if ("2".equals(currentLine)) {
+                        isBruteforce = true;
+                        currentLine = "exit";
+                        break;
+
+                    } else if ("3".equals(currentLine)) {
+                        isAnalysis = true;
+
+                        System.out.printf("Do you have an additional file with sample text from the same author?" +
+                                "%n1. Yes" +
+                                "%n2. No");
+                        while (!"exit".equals(currentLine)) {
+                            currentLine = scanner.nextLine();
+                            if ("1".equals(currentLine)) {
+                                while (!"exit".equals(currentLine)) {
+                                    System.out.println("Enter path for sample text file in txt format");
+                                    currentLine = scanner.nextLine();
+
+                                    if (Validator.isCorrectInPath(currentLine)) {
+                                        isAnalysisWithAddFile = true;
+                                        strPathInAddFile = currentLine;
+                                        currentLine = "exit";
+                                        break;
+                                    } else {
+                                        System.out.println("File not found or wrong format");
+                                        currentLine = scanner.nextLine();
+                                    }
+                                }
+                            }else if ("2".equals(currentLine)){
+                                currentLine = "exit";
+                                break;
+                            }
+                            else {
+                                System.out.println("Wrong answer");
+                            }
+                        }
+                    } else {
+                        System.out.println("Wrong answer");
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log